client/src/utils/persistence.js

```diff
@@ -0,0 +1,60 @@
+/**
+ * Handle localStorage persistence
+ */
+
+import { debounce } from 'lodash';
+import { getTranslations } from 'i18n';
+
+export const loadPersistedState = () => {
+  const state = {};
+  const language = localStorage.getItem('language');
```

Comment:
Perhaps merge to a single object that is stringified, that way you'd only have to getItem on something like a `settings` node.

Reply:
Fixed!